synfig-studio/src/gui/docks/dock_timetrack2.cpp

```diff
@@ -229,6 +230,34 @@ void Dock_Timetrack2::setup_tool_palette()
 		action_button_map[tool_button->get_name()] = tool_button;
 		tool_item_group->add(*tool_button);
 	}
+
+	Gtk::SeparatorToolItem* separator = Gtk::manage(new Gtk::SeparatorToolItem());
+	separator->set_valign(Gtk::ALIGN_CENTER);
+	tool_item_group->add(*separator);
+
+	struct InterpolationButtonInfo {
+		std::string name;
+		synfig::Interpolation interpolation;
+	};
+
+	const std::vector<InterpolationButtonInfo> interp_buttons_info{
+		{N_("Clamped"), synfig::INTERPOLATION_CLAMPED},
+		{N_("TCB"), synfig::INTERPOLATION_TCB},
+		{N_("Constant"), synfig::INTERPOLATION_CONSTANT},
+		{N_("Ease In/Out"), synfig::INTERPOLATION_HALT},
+		{N_("Linear"), synfig::INTERPOLATION_LINEAR}
+	};
+
+	for (const auto & interp_button_info: interp_buttons_info) {
+		Gtk::Image* image= Gtk::manage(new Gtk::Image());
+		image->set_from_icon_name(interpolation_icon_name(interp_button_info.interpolation), Gtk::IconSize::from_name("synfig-small_icon_16x16"));
+		Gtk::ToolButton *tool_button = manage(new Gtk::ToolButton(*image, _("interp_button_info.name")));

```

Comment:
`_("interp_button_info.name")` this is not right.
It will translate the string `"interp_button_info.name"` instead of the variable contents.

Use `_(interp_button_info.name)`

Reply:
oops my bad didn't notice.